builder/job.go

```diff
@@ -50,7 +50,7 @@ var validCommitCommands = map[string]bool{
 type Config struct {
 	DockerfileName string
 	RemoteURL      string
-	RepoName       string
+	RepoName       interface{}
```

Comment:
We don't want to use an `interface{}` here.
Entrpoiny/Cmd have some special parsing you can look at (in runconfig/config.go) to handle this.